lib/helper.js

```diff
@@ -5,6 +5,20 @@ const hoek = require('@hapi/hoek');
 const dayjs = require('dayjs');
 const TEMPLATE_NAME_REGEX_WITH_NAMESPACE = schema.config.regex.FULL_TEMPLATE_NAME_WITH_NAMESPACE;
 const formatDate = dateTime => dayjs(dateTime).format('YYYY-MM-DD');
+// SCM handles only SUCCESS, PENDING & FAILURE status
+const SCM_STATE_MAP = {
+    ABORTED: 'FAILURE',
+    CREATED: 'PENDING',
+    FAILURE: 'FAILURE',
+    QUEUED: 'PENDING',
+    RUNNING: 'PENDING',
+    SUCCESS: 'SUCCESS',
+    BLOCKED: 'PENDING',
+    UNSTABLE: 'PENDING',
+    COLLAPSED: '',
+    FIXED: '',
```

Comment:
this status is only sent to notifications, not a valid build status
https://github.com/screwdriver-cd/data-schema/blob/master/models/build.js#L14-L25

Reply:
But in the model it is defined as build status. Please check here:
https://github.com/screwdriver-cd/data-schema/blob/master/models/build.js#L113-L116

And, if it's not build status, then where to find only the build statuses?

Reply:
this is where all the build statuses are: https://github.com/screwdriver-cd/data-schema/blob/master/models/build.js#L14-L25